GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -910,6 +915,10 @@ extension ChatViewModel {
     ) -> Bool {
         return section.item(after: row) == nil
     }
+
+    func removePendingMessage(by messageId: String) {
+        pendingMessages.removeAll { $0.payload.messageId.rawValue.uppercased() == messageId.uppercased() }
```

Comment:
Are we sure that it is safe to alter messageId?

Reply:
@ykyivskyi-gl  Message ID, represented by `UUID` format, is case insensitive, but here it is uppercased for comparison. This approach is also used [here](https://github.com/salemove/ios-sdk-widgets/blob/4fab8cae530bb766337a1df54166eefe062d53b1/GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift#L406), so it is not something invented just now. Also message ID generation is something that we have control on Core SDK side, so I think it is safe to change message ID case for equality check. This is necessary because in some places message ID is represented as `Foundation.UUID` thus uppercased string, but in other places is is a raw `String` when received from backend and decoded as `String`.

Reply:
Got the idea, thanks for explaining!